AirCasting/AuthViews/SignInView.swift

```diff
@@ -51,8 +52,11 @@ private extension SignInView {
                     }
                     signinButton
                     signupButton
+                    Spacer()
                 }
                 .padding()
+                .navigationBarTitle("")
```

Comment:
Is it necessary? I dunno, but when I remove it nothing changes, that's why I'm asking :D